src/Sarif.WorkItems/SarifWorkItemFiler.cs

```diff
@@ -78,11 +78,12 @@ public virtual void FileWorkItems(Uri sarifLogFileLocation)
             throw new ArgumentException($"Specified URI was not an absolute file URI: {sarifLogFileLocation}");
         }
 
-        public virtual void FileWorkItems(string sarifLogFileContents)
+        public virtual void FileWorkItems(string sarifLogFileContents, out SarifLog sarifLog)
```

Comment:
I might be missing something here, but why an out parameter vs. making it the method return?  We overwrite the parameter without checking it almost immediately, so to me this is confusing as it gives the impression we may be using it in some way but, in fact, do not.  In this case I'd prefer to make it a returned value.